S2VX.Game/Story/Command/HoldNotesColorCommand.cs

```diff
@@ -6,8 +6,8 @@ public class HoldNotesColorCommand : S2VXCommand {
         public Color4 EndValue { get; set; } = S2VXColorConstants.Cyan;
         public override void Apply(double time, S2VXStory story) {
             var value = S2VXUtils.ClampedInterpolation(time, StartValue, EndValue, StartTime, EndTime, Easing);
-            story.Notes.GetHoldNotes().ForEach(note => note.SetColor(value));
             story.Notes.PreviewHoldNoteColor = value;
```

Comment:
I think we should just have `HoldNoteColor` and have the preview hold note pull from that value. Same with `PreviewNotecolor`.

Reply:
Implemented. I'll have the Preview note pull from the same constants for InnerColor, OutlineColor, and OutlineThickness. I'll leave Alpha alone so you can easily differentiate it from the notes already placed.